### PR TITLE
AR-890: Added config option for setting token TTL for screen users

### PR DESCRIPTION
**Pull request by turegjorup**
#### Link to ticket

https://jira.itkdev.dk/browse/AR-890

#### Description

Added config option for setting token TTL for screen users

#### Screenshot of the result

If your change affects the user interface you should include a screenshot of the result with the pull request.

#### Checklist

- [ ] My code is covered by test cases.
- [ ] My code passes our test (all our tests).
- [ ] My code passes our static analysis suite.
- [ ] My code passes our continuous integration process.

If your code does not pass all the requirements on the checklist you have to add a comment explaining why this change 
should be exempt from the list.

#### Additional comments or questions

If you have any further comments or questions for the reviewer please add them here.

### CODE REVIEW

File: src/Security/ScreenAuthenticator.php [hunk 1]
```diff
@@ -153,7 +153,7 @@ public function unbindScreen(Screen $screen): void
     private function generateBindKey(): string
     {
         $length = 8;
-        $chars = '123456789ABCDEFGHIJKLMNPQRSTUVWXYZ';
```
Comment (tuj) [inline src/Security/ScreenAuthenticator.php:156]: Why this change?

**Review by tuj — APPROVED** [review]

File: src/Security/ScreenAuthenticator.php (hunk 1 above)
Comment (turegjorup) [reply src/Security/ScreenAuthenticator.php:156]: Having just numbers vs. numbers and chars makes no real difference from a security perspective. But having (capital) letters in the code is an inconvenience when entering the code.
